cpp/src_prims/linalg/reduce_rows_by_key.cuh

```diff
@@ -171,7 +171,7 @@ void sum_rows_by_key_small_nkeys(const DataIteratorT d_A, int lda,
 
 //
 // Reduce by keys - large number of keys
-// Computing a "weigthed histogram" with local histograms in smem
+// Computing a "weig.cuhed histogram" with local histograms in smem
```

Comment:
this is probably the result of an automated regex application!? :)

Reply:
Oh lol, surely.  Initally I missed that `.` is a wildcard :(